dashboard/src/components/Common/Autocomplete/AutocompleteField.jsx

```diff
@@ -74,7 +91,8 @@ class AutocompleteField extends React.Component {
           className: `form-control ${this.props.className}`,
           value: fieldProps.value,
           placeholder: this.props.placeholder,
-          onChange: (event, { newValue }) => fieldProps.onChange(newValue) }}
+          onChange: (event, { newValue }) => fieldProps.onChange(newValue),
+          onKeyDown: (event) => this.tabCheck(event, suggestions, fieldProps)}}
```

Comment:
The calling pattern here makes the constructor's `this.tabCheck.bind` call redundant.